syntax = "proto3";

enum ByteOrder {
    LE = 0;
    BE = 1;
}

message Frame {
    fixed64 c_time = 1;
    fixed64 a_time = 2;
    bytes data = 3;
    uint32 bits_per_word = 4;
    ByteOrder byte_order = 5;
}
